tools/clang/tools/dotnetc/OptEditorForm.cs

```diff
@@ -71,8 +71,8 @@ private void LeftButton_CheckedChanged(object sender, EventArgs e)
         private void ApplyChangesButton_Click(object sender, EventArgs e)
```

Comment:
I do not think you actually changed this file. For the future readers of this commit, could you please drop the change of this file?